cpp/data/system_updates/102.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -o errexit
+
+
+if [[ $(hostname) != "ub16.uni-tuebingen.de" ]]; then
```

Comment:
If you want this to happen on ub16, it must also happen on its successor ub55.v264.uni-tuebingen.de.

And => is there any specific reason why we do not want the cronjob on the test servers? Why? (its just generating a list from Zeder anyway)

Reply:
I think we currently only need this on Krimdok systems and I have adjusted the script accordingly.

Reply:
OK but if we need it for krimdok, then why did you use ub16 before and not ub15? If you change this now, it is also necessary to move the cronjob installer definition from ixtheo.cronjobs to krimdok.cronjobs.